gensim/test/test_doc2vec.py

```diff
@@ -306,6 +306,16 @@ def testTrainWarning(self, l):
                 model.alpha += 0.05
         warning = "Effective 'alpha' higher than previous training cycles"
         self.assertTrue(warning in str(l))
+
+    @log_capture()
+    def testLoadOnClassWarning(self):
+        """Test if logger warning is raised when loading doc2vec model on instance"""
+        model = doc2vec.Doc2Vec(DocsLeeCorpus(), min_count=1)
+        model.save(testfile())
+        load_on_instance = doc2vec.Doc2Vec()
+        model = load_on_instance.load(testfile())
+        warning = "Load methods should be called on a class object."
+        self.assertTrue(warning in str(l))
```

Comment:
Unfortunately it seems to be the 'usual' name for the LogCapture objects provided by the testfixtures package `@log_capture` decorator. (See for example: https://testfixtures.readthedocs.io/en/latest/logging.html#the-decorator) But, that would seem to require an `l` parameter be added to the method, which I don't see here... so a bit baffled that this could work.